src/img/logos/je-logo.v3.svg

```diff
@@ -1 +1,5 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="98" height="24" viewBox="0 0 98 24">
```

Comment:
Can you remove the width and height from the SVG's? Also, if you can run it through SVGOMG to minify it down 👍